sqlalchemy_exasol/base.py

```diff
@@ -541,9 +541,10 @@ def _get_tables_for_schema_odbc(self, connection, odbc_connection, schema, table
     def get_table_names(self, connection, schema, **kw):
         odbc_connection = self.getODBCConnection(connection)
         if odbc_connection is not None and not self.use_sql_fallback(**kw):
-            return self.get_table_names_odbc(connection, odbc_connection, schema, **kw)
+            result=self.get_table_names_odbc(connection, odbc_connection, schema, **kw)
```

Comment:
Why was the `return` statement replaced with an extra variable?

Reply:
Likely, a leftover of some debugging, I am going to change it back.

Reply:
resolved by https://github.com/blue-yonder/sqlalchemy_exasol/pull/101/commits/a50d105848909781b015566912405e5397176c97